### ISSUE TITLE
How to record a screen or desktop using this tool?

### ISSUE THREAD

**Issue by GeraldEugenio** [cmxl/FFmpeg.NET#67]

**Comment by lostmsu** [comment]
Also interested in this.

Wanted to clarify if this question was left unanswered because the feature is not currently available (on Windows, it is done using `ffmpeg -f gdigrab -i desktop`), or because nobody bothered to answer?

@GeraldEugenio have you found any FFmpeg C# wrapper that supports screen recording?

**Comment by fritz-net** [comment]
https://github.com/cmxl/FFmpeg.NET/issues/7

does this answer your question?

**Comment by lostmsu** [comment]
Yep, but it turns out ffmpeg is very inefficient because gdigrab is an old gen tech. These days the desktop is in DWM DXGI buffers, and there's an efficient API in modern Windows SDK with less options but actually much more practical: https://github.com/MicrosoftDocs/SimpleRecorder/
